lib/iris/experimental/animate.py

```diff
@@ -41,7 +41,11 @@ def animate(cube_iterator, plot_func, fig=None, **kwargs):
         object. See :meth:`iris.cube.Cube.slices`.
 
     * plot_func (:mod:`~iris.plot` or :mod:`~iris.quickplot` plot):
-        Plotting function used to animate.
+        Plotting function used to animate. Must accept the signature 
```

Comment:
Remove the `~` from the front so that they render with full names, perhaps like this:

```
* plot_func (:mod:`iris.plot` or :mod:`iris.quickplot` plotting function)
```